clang-tidy-diff.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
-CLANG_DIR=$(dirname $(dirname $(which clang-tidy)))
+set -e
+CLANG_TIDY=$(realpath $(which clang-tidy))
+CLANG_DIR=$(dirname $(dirname $CLANG_TIDY))
 CLANG_TIDY_DIFF=$CLANG_DIR/share/clang/clang-tidy-diff.py
-MERGE_BASE=$(git merge-base master HEAD)
-TIDY_MSG=$(git diff -U0 $MERGE_BASE | $CLANG_TIDY_DIFF -quiet -p1 2> /dev/null)
+if [ ! -f $CLANG_TIDY_DIFF ]; then
+  echo "not found: $CLANG_TIDY_DIFF"
+  exit 1
```

Comment:
This is not necessarily an error in some configs in which `$(BUILD_SUBDIR)` is a subdirectory, such as `out`.